### DISCUSSION TITLE
クリスマスに雪を降らせてほしい⛄

### DISCUSSION THREAD

**Discussion by hellowork-mhlw** [increments/qiita-discussions#381] [要望・アイデア・不具合]
Twitterで誕生日になったら風船が飛ぶみたいに、Qiitaに雪が降るとおもしろいと思いました

**Comment by tanaka-asami415** [comment]
@hellowork-mhlw 
ご意見ありがとうございます🙏
今後はそういうものも少しずつ対応していきたいと思っております！
